doc/Dockerpool/SkipDeleteArchitectureNotes.md

```diff
@@ -0,0 +1,41 @@
+# Raw and rough version
+
+All scripts that are using the pipeline are using a docker-container
+created from a docker image of repairnator-pipeline. This implies that
+we only use pipeline through Docker. *Is this hardcoded, or just how we
+do it? Considering the skipDelete is a part of the repairnator config
+I'd assume it is hardcoded, but that perhaps the pipeline could be
+used without it*
+
+The image used is specified in repairnator.cfg, and is by default
+```surli/repairnator:latest```.
```

Comment:
Actually it changed: the default is no more on my personal docker hub repository but on the Spirals Team's one, so it's `spirals/repairnator:latest`

Reply:
Fixed.